shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/impl/DataSourcesQueryBackendHandler.java

```diff
@@ -42,6 +46,8 @@
     
     private Map<String, DataSourceParameter> dataSourceParameterMap;
     
+    private String schemaName;
```

Comment:
It is not good a design for use global field which assigned out of the constructor.
Can you consider about improve it?

Reply:
ok.